diff --git a/index.js b/index.js
index 3d0c3753e35af211fa996b83bd5f6638d89c64df..f1e4ed55debeb6c25dcbd78c4a66e8d07637ac32 100644
--- a/index.js
+++ b/index.js
@@ -121,6 +121,7 @@ export default function (opts = {}) {
 				// see https://github.com/sveltejs/kit/issues/16092
 				manualChunks(id) {
 					if (id.startsWith(server)) {
+						if (id === `${server}/chunks/env.js`) return;
 						return id.slice(server_path_length);
 					}
 				}
