diff --git a/index.js b/index.js
index 3d0c3753e35af211fa996b83bd5f6638d89c64df..175f9f9092ff60ba7d0636ec7a19d39800492baf 100644
--- a/index.js
+++ b/index.js
@@ -120,6 +120,14 @@ export default function (opts = {}) {
 				// a circular import chain
 				// see https://github.com/sveltejs/kit/issues/16092
 				manualChunks(id) {
+					if (
+						id === `${server}/chunks/env.js` ||
+						id === `${server}/chunks/EventDebugDialog.js` ||
+						id === `${server}/chunks/howler.js`
+					) {
+						return undefined;
+					}
+
 					if (id.startsWith(server)) {
 						return id.slice(server_path_length);
 					}
